--- a/Main_WStatus_Content.asp
+++ b/Main_WStatus_Content.asp
@@ -143,38 +143,38 @@
 }
 }
 if (dataarray24.length == 0) {
-document.getElementById('wifi24headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Wireless 2.4 GHz is disabled.</span>';
+document.getElementById('wifi24headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Sieć bezprzewodowa 2,4 GHz jest wyłączona.</span>';
 } else {
-display_header(dataarray24, 'Wireless 2.4 GHz', document.getElementById('wifi24headerblock'), []);
+display_header(dataarray24, 'Sieć bezprzewodowa 2,4 GHz', document.getElementById('wifi24headerblock'), []);
 display_clients(wificlients24, document.getElementById('wifi24block'), 0);
 }
 if (band6g_support) {
 if (dataarray6.length == 0) {
-document.getElementById('wifi6headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Wireless 6 GHz is disabled.</span>';
+document.getElementById('wifi6headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Sieć bezprzewodowa 6 GHz jest wyłączona.</span>';
 } else {
-display_header(dataarray6, 'Wireless 6 GHz', document.getElementById('wifi6headerblock'), []);
+display_header(dataarray6, 'Sieć bezprzewodowa 6 GHz', document.getElementById('wifi6headerblock'), []);
 display_clients(wificlients6, document.getElementById('wifi6block'), 2);
 }
 }
 if (band5g_support) {
 if (wl_info.band5g_2_support) {
 if (dataarray5.length == 0) {
-document.getElementById('wifi5headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Wireless 5 GHz-1 is disabled.</span>';
+document.getElementById('wifi5headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Sieć bezprzewodowa 5 GHz-1 jest wyłączona.</span>';
 } else {
-display_header(dataarray5, 'Wireless 5 GHz-1', document.getElementById('wifi5headerblock'), dfs_statusarray5);
+display_header(dataarray5, 'Sieć bezprzewodowa 5 GHz-1', document.getElementById('wifi5headerblock'), dfs_statusarray5);
 display_clients(wificlients5, document.getElementById('wifi5block'), 1);
 }
 if (dataarray52.length == 0) {
-document.getElementById('wifi52headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Wireless 5 GHz-2 is disabled.</span>';
+document.getElementById('wifi52headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Sieć bezprzewodowa 5 GHz-2 jest wyłączona.</span>';
 } else {
-display_header(dataarray52, 'Wireless 5 GHz-2', document.getElementById('wifi52headerblock'), dfs_statusarray52);
+display_header(dataarray52, 'Sieć bezprzewodowa 5 GHz-2', document.getElementById('wifi52headerblock'), dfs_statusarray52);
 display_clients(wificlients52, document.getElementById('wifi52block'), 2);
 }
 } else {
 if (dataarray5.length == 0) {
-document.getElementById('wifi5headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Wireless 5 GHz is disabled.</span>';
+document.getElementById('wifi5headerblock').innerHTML='<span class="hint-color" style="font-size: 125%;">Sieć bezprzewodowa 5 GHz jest wyłączona.</span>';
 } else {
-display_header(dataarray5, 'Wireless 5 GHz', document.getElementById('wifi5headerblock'), dfs_statusarray5);
+display_header(dataarray5, 'Sieć bezprzewodowa 5 GHz', document.getElementById('wifi5headerblock'), dfs_statusarray5);
 display_clients(wificlients5, document.getElementById('wifi5block'), 1);
 }
 }
@@ -188,16 +188,16 @@
 var guestheader = 0;
 code = '<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3" class="FormTable">';
 code += '<thead><tr>';
-code += '<td width="25%">Device</td>';
-code += '<td width="29%">IP Address</td>';
-code += '<td width="16%">Rx/Tx & RSSI</td><td width="10%">Connected</td>';
+code += '<td width="25%">Urządzenie</td>';
+code += '<td width="29%">Adres IP</td>';
+code += '<td width="16%">Rx/Tx i RSSI</td><td width="10%">Połączono</td>';
 if (clientsarray.length > 1) {
 if (clientsarray[0][8] != "")
-code += '<td width="10%">Streams</td>';
+code += '<td width="10%">Strumienie</td>';
 else
 code += '<td width="10%">PHY</td>';
 }
-code += '<td width="10%">Flags</td>';
+code += '<td width="10%">Flagi</td>';
 code += '</tr></thead>';
 if (clientsarray.length > 1) {
 for (i = 0; i < clientsarray.length-1; ++i) {
@@ -269,7 +269,7 @@
 code += '</tr>';
 }
 } else {
-code += '<tr><td colspan="7">No clients</td></tr>';
+code += '<tr><td colspan="7">Brak klientów</td></tr>';
 }
 code += '</tr></table>';
 obj.innerHTML = code;
@@ -285,7 +285,7 @@
 code += '<td><span class="hint-color">Mode: </span>' + dataarray[7] + '</td>';
 code += '</tr>';
 code += '<tr>';
-code += '<td><span class="hint-color">Channel: </span>'+ dataarray[4] + '</td>';
+code += '<td><span class="hint-color">Kanał: </span>'+ dataarray[4] + '</td>';
 if (dataarray[1] != 0)
 code += '<td><span class="hint-color">RSSI: </span>' + dataarray[1] + ' dBm</td>';
 else
@@ -293,16 +293,16 @@
 if (dataarray[2] != 0)
 code += '<td><span class="hint-color">SNR: </span>' + dataarray[2] +' dB</td>';
 if (dataarray[3] != 0)
-code += '<td><span class="hint-color">Noise: </span>' + dataarray[3] + ' dBm</td>';
+code += '<td><span class="hint-color">Szum: </span>' + dataarray[3] + ' dBm</td>';
 if (dataarray[6] != -1)
-code += '<td><span class="hint-color">Utilization: </span>'+ dataarray[6] + '%</td>';
+code += '<td><span class="hint-color">Wykorzystanie: </span>'+ dataarray[6] + '%</td>';
 code += '</tr>';
 if (dfs_statusarray.length > 1) {
-code += '<tr><td colspan="2"><span class="hint-color">DFS State: </span>' + dfs_statusarray[0] + '</td>';
+code += '<tr><td colspan="2"><span class="hint-color">Stan DFS: </span>' + dfs_statusarray[0] + '</td>';
 time = parseInt(dfs_statusarray[1]);
 formatted_time = Math.floor(time / 3600) + "h " + Math.floor(time / 60) % 60 + "m " + time % 60 + "s";
-code += '<td><span class="hint-color">Time elapsed: </span>' + formatted_time + '</td>';
-code += '<td><span class="hint-color">Channel cleared for radar: </span>' + dfs_statusarray[2] + '</td></tr>';
+code += '<td><span class="hint-color">Czas, który upłynął: </span>' + formatted_time + '</td>';
+code += '<td><span class="hint-color">Kanał sprawdzony pod kątem radaru: </span>' + dfs_statusarray[2] + '</td></tr>';
 }
 code += '</table>';
 obj.innerHTML = code;
@@ -378,13 +378,13 @@
 <div>&nbsp;</div>
 <div class="formfonttitle"><#873#> - <#527#></div>
 <div style="margin:10px 0 10px 5px;" class="splitLine"></div>
-<div class="formfontdesc">List of connected Wireless clients</div>
+<div class="formfontdesc">Lista podłączonych klientów sieci bezprzewodowej</div>
 <table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3" class="FormTable">
 <tr>
-<th>Refresh frequency</th>
+<th>Częstotliwość odświeżania</th>
 <td>
 <select name="refreshrate" class="input_option" onchange="setRefresh(this);" id="refreshrate">
-<option value="0">No refresh</option>
+<option value="0">Bez odświeżania</option>
 <option value="1">1 second</option>
 <option value="3" selected>3 seconds</option>
 <option value="5">5 seconds</option>
@@ -393,9 +393,9 @@
 </td>
 </tr>
 <tr>
-<th>Display low level details</th>
+<th>Pokaż szczegóły niskiego poziomu</th>
 <td>
-<input class="button_gen" type="button" onclick="open_details_window();" value="Open">
+<input class="button_gen" type="button" onclick="open_details_window();" value="Otwórz">
 </td>
 </tr>
 </table>
@@ -411,8 +411,8 @@
 <br><br>
 <div id="wifi6headerblock"></div>
 <div id="wifi6block"></div>
-<div id="flags_mumimo_div" style="display:none;">Flags: <span class="hint-color">P</span>=Powersave Mode, <span class="hint-color">S</span>=Short GI, <span class="hint-color">T</span>=STBC, <span class="hint-color">M</span>=MU Beamforming, <span class="hint-color">A</span>=Associated, <span class="hint-color">U</span>=Authenticated</div>
-<div id="flags_div">Flags: <span class="hint-color">P</span>=Powersave Mode, <span class="hint-color">S</span>=Short GI, <span class="hint-color">T</span>=STBC, <span class="hint-color">A</span>=Associated, <span class="hint-color">U</span>=Authenticated</div>
+<div id="flags_mumimo_div" style="display:none;">Flagi: <span class="hint-color">P</span>=tryb oszczędzania energii, <span class="hint-color">S</span>=krótki GI, <span class="hint-color">T</span>=STBC, <span class="hint-color">M</span>=MU Beamforming, <span class="hint-color">A</span>=powiązany, <span class="hint-color">U</span>=uwierzytelniony</div>
+<div id="flags_div">Flagi: <span class="hint-color">P</span>=tryb oszczędzania energii, <span class="hint-color">S</span>=krótki GI, <span class="hint-color">T</span>=STBC, <span class="hint-color">A</span>=powiązany, <span class="hint-color">U</span>=uwierzytelniony</div>
 <br>
 <div class="apply_gen">
 <input type="button" onClick="location.reload();" value="<#1828#>" class="button_gen" >
