--- a/Main_IPTStatus_Content.asp
+++ b/Main_IPTStatus_Content.asp
@@ -27,16 +27,16 @@
 var code, i, line;
 var Hours, Minutes, Seconds;
 code = '<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3" class="FormTable_table">';
-code += '<thead><tr><td colspan="6">UPnP IGD &amp; PCP/NAT-PMP IPv4 port forwards</td></tr></thead>';
-code += '<tr><th width="8%">Proto</th>';
-code += '<th width="8%">Ext Port</th>';
-code += '<th width="17%">Local IP</th>';
-code += '<th width="12%">Local Port</th>';
-code += '<th width="13%">Time left</th>';
-code += '<th width="42%">Description</th>';
+code += '<thead><tr><td colspan="6">Przekierowania portów IPv4 UPnP IGD i PCP/NAT-PMP</td></tr></thead>';
+code += '<tr><th width="8%">Protokół</th>';
+code += '<th width="8%">Port zewnętrzny</th>';
+code += '<th width="17%">Lokalny adres IP</th>';
+code += '<th width="12%">Port lokalny</th>';
+code += '<th width="13%">Pozostały czas</th>';
+code += '<th width="42%">Opis</th>';
 code += '</tr>';
 if ("<% nvram_get("wan_upnp_enable"); %>" == "0") {
-code += '<tr><td colspan="6"><span>Service is currently disabled. Can be enabled <a href="Advanced_WAN_Content.asp" style="text-decoration: underline;">here</a>.</span></td></tr>';
+code += '<tr><td colspan="6"><span>Usługa jest obecnie wyłączona. Można ją włączyć <a href="Advanced_WAN_Content.asp" style="text-decoration: underline;">tutaj</a>.</span></td></tr>';
 } else {
 if (upnparray.length > 1) {
 for (i = 0; i < upnparray.length-1; ++i) {
@@ -58,7 +58,7 @@
 code += '</tr>';
 }
 } else {
-code += '<tr><td colspan="6"><span>No active port forwards.</span></td></tr>';
+code += '<tr><td colspan="6"><span>Brak aktywnych przekierowań portów.</span></td></tr>';
 }
 }
 code += '</tr></table>';
@@ -67,13 +67,13 @@
 function show_vserver() {
 var code, i, line;
 code = '<table width="100%" border="1" align="center" cellpadding="4" cellspacing="0" bordercolor="#6b8fa3" class="FormTable_table">';
-code += '<thead><tr><td colspan="6">Virtual Servers</td></tr></thead>';
-code += '<tr><th width="20%">Source IP</th>';
-code += '<th width="10%">Proto</th>';
-code += '<th width="15%">Ext Port range</th>';
-code += '<th width="20%">Local IP</th>';
-code += '<th width="15%">Local Port</th>';
-code += '<th width="20%">Chain</th>';
+code += '<thead><tr><td colspan="6">Serwery wirtualne</td></tr></thead>';
+code += '<tr><th width="20%">Źródłowy adres IP</th>';
+code += '<th width="10%">Protokół</th>';
+code += '<th width="15%">Zakres portów zewnętrznych</th>';
+code += '<th width="20%">Lokalny adres IP</th>';
+code += '<th width="15%">Port lokalny</th>';
+code += '<th width="20%">Łańcuch</th>';
 code += '</tr>';
 if (vserverarray.length > 1) {
 for (i = 0; i < vserverarray.length-1; ++i) {
@@ -88,7 +88,7 @@
 code += '</tr>';
 }
 } else {
-code += '<tr><td colspan="6"><span>No active port forwards.</span></td></tr>';
+code += '<tr><td colspan="6"><span>Brak aktywnych przekierowań portów.</span></td></tr>';
 }
 code += '</tr></table>';
 document.getElementById("vserverblock").innerHTML = code;
