--- a/router.asp
+++ b/router.asp
@@ -412,7 +412,7 @@
 }
 code = '';
 if(system.INTELplatform || system.modelName == 'RT-AC87U'){
-code += '<div id="wl_ready" class="wl-ready" style="display:none;">Wireless is setting...</div>';
+code += '<div id="wl_ready" class="wl-ready" style="display:none;">Trwa konfigurowanie sieci bezprzewodowej...</div>';
 }
 for(var i=0; i<wlInterface.length; i++){
 var unit = wlInterface[i][2];
@@ -504,10 +504,10 @@
 code += '</div>';
 }
 code += '<div id="qrcodepanel' + unit + '" class="qrcodepanel" style="display:none; width:180px; margin-top:-200px; margin-left:0px;">';
-code += '<div style="padding:10px;"><div style="text-align:center;">Scan to connect:</div>';
+code += '<div style="padding:10px;"><div style="text-align:center;">Zeskanuj, aby połączyć:</div>';
 code += '<div style="margin:10px 0 10px 0px;height:2px;width:100%;padding:0;" class="splitLine"></div>';
-code += '<div class="qrcodepanelpad" id="qr' + unit +'"></div><input style="margin-top:10px; width:100%;" type="button" class="button_gen" value="Close" onclick="hide_qr_code(' + unit + ');"></div></div>';
-code += '<div style="display:none;" id="showqrdiv' + unit + '"><span style="color:#FFCC00;cursor:pointer;text-decoration:underline" onclick="show_qr_code(' + unit + ');">Show QR code</span></div>';
+code += '<div class="qrcodepanelpad" id="qr' + unit +'"></div><input style="margin-top:10px; width:100%;" type="button" class="button_gen" value="Zamknij" onclick="hide_qr_code(' + unit + ');"></div></div>';
+code += '<div style="display:none;" id="showqrdiv' + unit + '"><span style="color:#FFCC00;cursor:pointer;text-decoration:underline" onclick="show_qr_code(' + unit + ');">Pokaż kod QR</span></div>';
 code += '</div>';
 }
 $('#wl_settings_field').html(code);
